pkg/bmc/redfish_virtualmedia.go

```diff
@@ -89,3 +89,8 @@ func (a *redfishVirtualMediaAccessDetails) VendorInterface() string {
 func (a *redfishVirtualMediaAccessDetails) SupportsSecureBoot() bool {
 	return true
 }
+
+func (a *redfishVirtualMediaAccessDetails) Validate(bmcCreds Credentials) (bool, error) {
+	Address := getRedfishAddress(a.bmcType, a.host)
+	return ValidateVirtualMedia(Address, bmcCreds.Username, bmcCreds.Password, a.disableCertificateVerification)
```

Comment:
Nit, should be lowercase

```suggestion
	address := getRedfishAddress(a.bmcType, a.host)
	return ValidateVirtualMedia(address, bmcCreds.Username, bmcCreds.Password, a.disableCertificateVerification)
```

Reply:
Done